### PR TITLE
fix(#minor); perpfutures-sdk; fix pool snapshots, snapshot daily activity

### PR DESCRIPTION
**Pull request by dhruv-chauhan**

### CODE REVIEW

File: subgraphs/_reference_/src/sdk/protocols/perpfutures/poolSnapshot.ts [hunk 1]
```diff
@@ -458,35 +458,20 @@ export class PoolSnapshot {
     snapshot.cumulativeUniqueUsers = this.pool.cumulativeUniqueUsers;
 
     const dailyActivityHelper = initActivityHelper(
-      Bytes.fromUTF8("daily-".concat(this.dayID.toString()))
+      Bytes.fromUTF8("daily-".concat(day.toString()))
     );
     snapshot.dailyActiveUsers = dailyActivityHelper.activeUsers;
+    snapshot.dailyActiveDepositors = dailyActivityHelper.activeDepositors;
```
Comment (dmelotik) [inline subgraphs/_reference_/src/sdk/protocols/perpfutures/poolSnapshot.ts:464]: Good catch on this!

**Review by dmelotik — APPROVED** [review]
Good catch!!

@bye43 we should update any perpfuture subgraph using the sdk with this. Otherwise they will not have accurate `dailyActiveDepositors` etc

**Comment by dmelotik** [comment]
I will make an issue to track this @dhruv-chauhan, but you should also have permissions if you find any bugs in the future.

#2056

**Comment by bye43** [comment]
cc: @ishraq8 for visibility

**Pull request merged**